packages/luigis-box/src/Resources/config/graphql-types/RecommendationType.types.yaml

```diff
@@ -0,0 +1,14 @@
+RecommendationType:
```

Comment:
I am afraid this needs to be a decorator, too, to enable the extension from the project, or am I missing something?